lib/money/null_currency.rb

```diff
@@ -28,7 +28,11 @@ def eql?(other)
     def to_s
       ''
     end
-    
+
+    def no_currency?
```

Comment:
`no_currency?` sounds pretty vague and not clear what it's used for, can we name this method something else? Like don't overwrite, don't write or read only

Reply:
`@money.no_currency?` works to me. `@currency.no_currency?` doesn't work though. Will change to only keep the one in money. Does that work for you?